### PR TITLE
Testbranch

### PR DESCRIPTION
**Pull request by takemikami**
<a href='#crh-start'></a><a href='#crh-data-%7B%22approved%22%3A%20%7B%22https%3A//github.com/fs-mikami%22%3A%20%7B%22avatar_url%22%3A%20%22https%3A//avatars.githubusercontent.com/u/9067199%3Fv%3D3%22%7D%7D%2C%20%22comments%22%3A%20%7B%22General%20Comment%22%3A%20%7B%22html_url%22%3A%20%22https%3A//github.com/takemikami/sandbox/pull/2%23issuecomment-150070874%22%2C%20%22comments%22%3A%20%5B%7B%22body%22%3A%20%22%3Ashipit%3A%22%2C%20%22created_at%22%3A%20%222015-10-22T01%3A24%3A07Z%22%2C%20%22user%22%3A%20%7B%22avatar_url%22%3A%20%22https%3A//avatars.githubusercontent.com/u/9067199%3Fv%3D3%22%2C%20%22html_url%22%3A%20%22https%3A//github.com/fs-mikami%22%7D%7D%5D%2C%20%22title%22%3A%20%22General%20Comment%22%7D%2C%20%22Commit%2014c991802d82f3b8acc3c4a4a96ea2737969a87e%20hoge%205%205%22%3A%20%7B%22html_url%22%3A%20%22https%3A//github.com/takemikami/sandbox/commit/14c991802d82f3b8acc3c4a4a96ea2737969a87e%23commitcomment-13918324%22%2C%20%22comments%22%3A%20%5B%7B%22body%22%3A%20%22%5Cu3066%5Cu3059%5Cu3068%22%2C%20%22created_at%22%3A%20%222015-10-22T05%3A54%3A01Z%22%2C%20%22user%22%3A%20%7B%22avatar_url%22%3A%20%22https%3A//avatars.githubusercontent.com/u/1129652%3Fv%3D3%22%2C%20%22html_url%22%3A%20%22https%3A//github.com/takemikami%22%7D%7D%5D%2C%20%22title%22%3A%20%22File%3A%20hoge%3AL5%20%2814c9918%29%22%7D%2C%20%22Pull%2014c991802d82f3b8acc3c4a4a96ea2737969a87e%20hoge%204%22%3A%20%7B%22html_url%22%3A%20%22https%3A//github.com/takemikami/sandbox/pull/2%23discussion_r42713225%22%2C%20%22comments%22%3A%20%5B%7B%22body%22%3A%20%22good%21%22%2C%20%22created_at%22%3A%20%222015-10-22T05%3A46%3A45Z%22%2C%20%22user%22%3A%20%7B%22avatar_url%22%3A%20%22https%3A//avatars.githubusercontent.com/u/1129652%3Fv%3D3%22%2C%20%22html_url%22%3A%20%22https%3A//github.com/takemikami%22%7D%7D%5D%2C%20%22title%22%3A%20%22File%3A%20hoge%3AL1-7%22%7D%2C%20%22Commit%2053da6b0fbad4d4f04cf07e09bb46a84370b43713%22%3A%20%7B%22html_url%22%3A%20%22https%3A//github.com/takemikami/sandbox/commit/53da6b0fbad4d4f04cf07e09bb46a84370b43713%23commitcomment-13916107%22%2C%20%22comments%22%3A%20%5B%7B%22body%22%3A%20%22test%22%2C%20%22created_at%22%3A%20%222015-10-22T01%3A25%3A19Z%22%2C%20%22user%22%3A%20%7B%22avatar_url%22%3A%20%22https%3A//avatars.githubusercontent.com/u/9067199%3Fv%3D3%22%2C%20%22html_url%22%3A%20%22https%3A//github.com/fs-mikami%22%7D%7D%2C%20%7B%22body%22%3A%20%22%3A%2B1%3A%22%2C%20%22created_at%22%3A%20%222015-10-22T01%3A26%3A24Z%22%2C%20%22user%22%3A%20%7B%22avatar_url%22%3A%20%22https%3A//avatars.githubusercontent.com/u/1129652%3Fv%3D3%22%2C%20%22html_url%22%3A%20%22https%3A//github.com/takemikami%22%7D%7D%5D%2C%20%22title%22%3A%20%22Commit%2053da6b0%20comments%22%7D%7D%2C%20%22processed%22%3A%20%5B%22https%3A//github.com/takemikami/sandbox/pull/2%23issuecomment-150070874%22%2C%20%22https%3A//github.com/takemikami/sandbox/commit/53da6b0fbad4d4f04cf07e09bb46a84370b43713%23commitcomment-13916107%22%2C%20%22https%3A//github.com/takemikami/sandbox/commit/53da6b0fbad4d4f04cf07e09bb46a84370b43713%23commitcomment-13916123%22%2C%20%22https%3A//github.com/takemikami/sandbox/pull/2%23discussion_r42713225%22%2C%20%22https%3A//github.com/takemikami/sandbox/commit/14c991802d82f3b8acc3c4a4a96ea2737969a87e%23commitcomment-13918324%22%5D%7D'></a>
<a href='https://www.codereviewhub.com/'></a>

<a href='https://github.com/fs-mikami'></a>
- [ ] <a href='#crh-comment-Pull 14c991802d82f3b8acc3c4a4a96ea2737969a87e hoge 4'></a> &nbsp;<b><a href='https://github.com/takemikami/sandbox/pull/2#discussion_r42713225'>File: hoge:L1-7</a></b>
- <a href='https://github.com/takemikami'></a> good!
- [ ] <a href='#crh-comment-Commit 14c991802d82f3b8acc3c4a4a96ea2737969a87e hoge 5 5'></a> &nbsp;<b><a href='https://github.com/takemikami/sandbox/commit/14c991802d82f3b8acc3c4a4a96ea2737969a87e#commitcomment-13918324'>File: hoge:L5 (14c9918)</a></b>
- <a href='https://github.com/takemikami'></a> てすと
- [ ] <a href='#crh-comment-Commit 53da6b0fbad4d4f04cf07e09bb46a84370b43713'></a> &nbsp;<b><a href='https://github.com/takemikami/sandbox/commit/53da6b0fbad4d4f04cf07e09bb46a84370b43713#commitcomment-13916107'>Commit 53da6b0 comments</a></b>
- <a href='https://github.com/fs-mikami'></a> test

<a href='https://www.codereviewhub.com/takemikami/sandbox/pull/2?mark_as_completed=1'></a>&nbsp;<a href='https://www.codereviewhub.com/takemikami/sandbox/pull/2?approve=1'></a>&nbsp;<a href='https://www.codereviewhub.com/takemikami/sandbox/pull/2?approve=0'></a>&nbsp;<a href='https://github.com/takemikami/sandbox/pull/2'></a>
<a href='#crh-end'></a>

### CODE REVIEW

**Comment by takemikami** [comment]
This pull request is now managed with CodeReviewHub. <a href='https://github.com/takemikami/sandbox/pull/2'>Refresh</a>.<a href='#crh-update-none'></a>

**Comment by fs-mikami** [comment]
:shipit:

File: hoge [hunk 1]
```diff
@@ -0,0 +1,6 @@
+a
+
+
+
```
Comment (takemikami) [inline hoge]: good!

**Comment by takemikami** [comment]
resolve #3

**Pull request closed without merging**